packages/mobile/src/invite/saga.ts

```diff
@@ -233,16 +235,38 @@ export function* doRedeemInvite(action: RedeemInviteAction) {
     // Add temp wallet so we can send money from it
     let tempAccount
     try {
-      // @ts-ignore
-      tempAccount = yield call(web3.eth.personal.importRawKey, String(inviteCode).slice(2), TEMP_PW)
+      if (isZeroSyncMode()) {
```

Comment:
Heads up, I reorganized this file yesterday, sorry for the conflict

Reply:
Thanks for the heads up. I will rebase soon.